src/main/java/com/actiontech/dble/plan/optimizer/JoinStrategyChooser.java

```diff
@@ -149,25 +150,14 @@ private void buildNestLoop(JoinNode joinNode, PlanNode node, boolean innerJoin)
         if (isSmallTable((TableNode) node) || canDoAsMerge(joinNode) || !innerJoin) {
             return;
         }
-        List<ItemFuncEqual> joinFilter = joinNode.getJoinFilter();
-        for (ItemFuncEqual itemFuncEqual : joinFilter) {
-            List<Item> arguments = itemFuncEqual.arguments();
-            Item item = arguments.stream().filter(argument -> !StringUtil.equals(getTableName((TableNode) node), argument.getTableName())).findFirst().get();
-            PlanNode dependNode = nodeMap.get(item.getTableName());
-            if (isSmallTable((TableNode) dependNode) && innerJoin) {
-                joinNode.setStrategy(Strategy.ALWAYS_NEST_LOOP);
-                node.setNestLoopFilters(new ArrayList<>());
-                node.setNestLoopDependNode(dependNode);
-                List<PlanNode> nodeList = Optional.ofNullable(dependNode.getNestLoopDependOnNodeList()).orElse(new ArrayList<>());
-                nodeList.add(nodeList.size(), node);
-                dependNode.setNestLoopDependOnNodeList(nodeList);
-                return;
-            }
-        }
         joinNode.setStrategy(JoinNode.Strategy.ALWAYS_NEST_LOOP);
+        PlanNode dependNode = findDependNode(node, innerJoin);
```

Comment:
the 'dependNode' naming does not meet the variable purpose. use dependedNode instead .

Reply:
get